addons/discuss-button/addon.json

```diff
@@ -86,14 +95,20 @@
       "id": "compact-nav",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Stick to top of screen",
+      "id": "stick",
+      "type": "boolean",
+      "default": true
```

Comment:
The default should probably be `false` here, unless I'm misunderstanding what the setting does

Reply:
its true by default cause if its false it enables the setting

Reply:
Oh, I see.
Generally, enabling an addon/setting replaces vanilla Scratch behavior with a custom one. Here, it does the opposite. I'd say it's a bit confusing.

Reply:
ye it was originally disabled by default but dnin wanted it like that so i just went with it

Reply:
> Generally, enabling an addon/setting replaces vanilla Scratch behavior with a custom one. Here, it does the opposite. I'd say it's a bit confusing.

Negative switches ("Don't stick to top of screen") can also be confusing.

Reply:
> ye it was originally disabled by default but dnin wanted it like that so i just went with it

Yeah, I switched it around because of the fact that you turn something on to disable something. But then it sounds like something that should be off by default, but that state is different than vanilla Scratch.

So how should we handle that?

Reply:
Should this be a two-choice single-select setting with the following options?

Stick to: `top of screen` `top of page`

Reply:
> Should this be a two-choice single-select setting with the following options?
> 
> Stick to: `top of screen` `top of page`

ye i think this is the best way to do it

Reply:
I don't think there's a good way to phrase "don't stick to the top" without using negatives. In that case, I guess a two-choice select setting would make sense

Reply:
I'll work on adding that.